src/Text/Pandoc/ImageSize.hs

```diff
@@ -451,3 +495,32 @@ webpSize opts img =
   case AW.parseOnly pWebpSize img of
     Left _   -> Nothing
     Right sz -> Just sz { dpiX = fromIntegral $ writerDpi opts, dpiY = fromIntegral $ writerDpi opts}
+
+imageTransform :: ByteString -> ImageTransform
```

Comment:
`imageTransform` seems too general a name for a function that just operates on exif.
Also, all exported functions and types should have Haddock comments.